.github/workflows/ci.yml

```
@@ -38,25 +38,16 @@ jobs:
     - name: Install C libraries for tooling on macOS
       if: matrix.os == 'macOS-latest'
       run: brew install libusb
-    - name: Check that all crates build
-      run: RUSTFLAGS='--deny warnings' cargo check --all
+    - name: Check that all crates that can be compiled for the host build
+      run: RUSTFLAGS='--deny warnings' cargo check --all --workspace
```

These flags mean the same thing, but `--all` is soft-deprecated

```suggestion
      run: RUSTFLAGS='--deny warnings' cargo check --workspace
```

---

I totally misread that and though it said `--all-targets` so I kept it but yeah if it's the same as --workspace it can go away.